code/modules/reagents/chemistry/reagents/drug_reagents.dm

```diff
@@ -457,22 +457,18 @@
 	metabolization_rate = 2 * REAGENTS_METABOLISM
 	overdose_threshold = 30
 
-/datum/reagent/drug/pumpup/on_mob_metabolize(mob/living/L)
-	..()
-	ADD_TRAIT(L, TRAIT_STUNRESISTANCE, type)
-
-/datum/reagent/drug/pumpup/on_mob_end_metabolize(mob/living/L)
-	REMOVE_TRAIT(L, TRAIT_STUNRESISTANCE, type)
-	..()
-
 /datum/reagent/drug/pumpup/on_mob_life(mob/living/carbon/M)
 	M.Jitter(5)
-
+	M.adjustStaminaLoss (-5, 0)
+	
 	if(prob(5))
 		to_chat(M, "<span class='notice'>[pick("Go! Go! GO!", "You feel ready...", "You feel invincible...")]</span>")
-	if(prob(15))
+	if(prob(30))
```

Comment:
why nerf it ? no one is gonna use it for 5 stam heal

Reply:
Because it is on par with most every other stamina regen chem except aranesp, and making it higher would make it just maint aranesp. If I didn't make it risky by making it cause heart damage, it would make it far too easy to stack with other chems like aranesp.

Reply:
for what it gives its better to just go to chem and make real aranesp, why do we need another stamina healer that no one will use? most "stamina healers" are just garbage flavourtown

Reply:
good thing chemists are well known for sharing their department so you can make combat chems.

Reply:
combat chems are garbage nerf them all into the ground so that we can eventually remove them for being useless

Reply:
I disagree. I think combat chems can be good if they have interesting effects and aren't able to be easily combined together, if at all. Obviously I don't want to argue about the differences of combat chems in a baton pr but I didn't want to leave pump up useless in its current state. Reworking the drugs can come later.

Reply:
> good thing chemists are well known for sharing their department so you can make combat chems.

a white jumpsuit and a medkit in hand will fool most idiots, and after medics got chem access territorial chemists get dunked before you will notice it

Reply:
Actually yeah now that most stuns are gone combat chems might not be shit